website/docs/components/table/partials/code/component-api.md

```diff
@@ -1,49 +1,107 @@
-    If defined, indicates which column should be pre-sorted when the table is rendered.
+    If defined, the value should be set to the key of the column that should be pre-sorted when the table is rendered.
   </C.Property>
   <C.Property @name="sortOrder" @type="string" @values={{array "asc" "desc" }} @default="asc">
     Use in conjunction with `sortBy`. If defined, indicates which direction the column should be pre-sorted in. If not defined, `asc` is applied by default.
   </C.Property>
-  <C.Property @name="onSort" @type="function">
-    Callback function invoked (if provided) when one of the sortable table headers is clicked, and a sorting is triggered. The function receives the values of `sortBy` and `sortOrder` as arguments.
-  </C.Property>
   <C.Property @name="isStriped" @type="boolean" @values={{array "false" "true" }} @default="false">
     Define on the table invocation. If set to `true`, row striping ("zebra striping") will be applied to the table.
   </C.Property>
-  <C.Property @name="isFixed" @type="boolean" @values={{array "false" "true" }} @default="false">
+  <C.Property @name="isFixedLayout" @type="boolean" @values={{array "false" "true" }} @default="false">
     If set to `true`, the `table-display`(CSS) property will be set to `fixed`, which will automatically distribute columns equally based on the total width of the table.
   </C.Property>
   <C.Property @name="density" @type="enum" @values={{array "short" "medium" "tall" }} @default="medium">
     If set, determines the density, or height, of the table’s rows.
   </C.Property>
+  <C.Property @name="align" @type="enum" @values={{array "left" "center" "right" }} @default="left">
+    If set, determines the text alignment for all cell (`td`) content in a table.
+  </C.Property>
   <C.Property @name="valign" @type="enum" @values={{array "top" "middle" }} @default="top">
     If set, determines the vertical alignment for all cell (`td`) content in a table. Does not apply to table headers (`th`).
   </C.Property>
   <C.Property @name="caption" @type="string">
-    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text.
+    Adds a (non-visible) caption for users with assistive technology. If set on a sortable table, the provided table caption is paired with the automatically generated sorted message text. Note: it is best practice to provide a table caption for users with assistive technology.
   </C.Property>
   <C.Property @name="...attributes">
     Supported for the `Hds::Table` component.
   </C.Property>
+  <C.Property @name="onSort" @type="function">
+    Automatically applied to sortable table headers, `onSort` is the callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
```

Comment:
As reader, I would not be sure what "Automatically applied to sortable table headers" exactly means.

```suggestion
    Callback function that is invoked when one of the sortable table headers is clicked (or has a keyboard interaction performed). The function receives the values of `sortBy` and `sortOrder` as arguments.
```